packages/message-backend/src/index.ts

```diff
@@ -0,0 +1,3 @@
+//export * from './service'
```

Comment:
Should maybe remove this as not needed atm. As well as the empty service file.

Reply:
Going to use this service in KYV-188